app/src/main/java/fr/neamar/kiss/forwarder/InterfaceTweaks.java

```diff
@@ -119,10 +119,21 @@ private void applyRoundedCorners(MainActivity mainActivity) {
                 mainActivity.findViewById(R.id.searchEditLayout).setBackgroundResource(R.drawable.rounded_search_bar);
             } else {
                 // Before API21, you can't access values from current theme using ?attr/
-                // So we made two different drawables (#931).
+                // So we made different drawable for each theme (#931).
+                Resources res = mainActivity.getResources();
+
                 if (getSearchBackgroundColor() == Color.WHITE) {
                     mainActivity.findViewById(R.id.externalFavoriteBar).setBackgroundResource(R.drawable.rounded_search_bar_pre21_light);
                     mainActivity.findViewById(R.id.searchEditLayout).setBackgroundResource(R.drawable.rounded_search_bar_pre21_light);
+                } else if (getSearchBackgroundColor() == Color.parseColor("#"+Integer.toHexString(res.getColor(R.color.kiss_background_light_transparent)))) {
```

Comment:
I think `getSearchBackgroundColor() == res.getColor(R.color.kiss_background_light_transparent) ` is enough no? They both return an int

Reply:
yep works, thanks!